PowerPlant.cs

```
@@ -0,0 +1,113 @@
+using System;
+
+namespace LambdaCore
+{
+    public class PowerPlant
+    {
+        private List<Core> cores = new List<Core>();
+        private Core selectedCore = null;
+
+        public void CreateCore(string type, int durability)
+        {
+            char coreName = (char)('A' + cores.Count());
+
+            if (type == "System")
+            {
+                Core core = new Core(TypeCore.SystemCore, coreName.ToString(), durability);
+                cores.Add(core);
+                Console.WriteLine($"Successfully created Core {coreName}!");
+            }
+            else if (type == "Para")
+            {
+                Core core = new Core(TypeCore.ParaCore, coreName.ToString(), durability);
+                cores.Add(core);
+                Console.WriteLine($"Successfully created Core {coreName}!");
+            }
+            else
+            {
+                throw new ArgumentException();
+            }
```

This can be moved at the top - if an invalid type is passed you can throw an InvalidCoreTypeException